src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -245,7 +245,9 @@ import Tooltip from '@nextcloud/vue/dist/Directives/Tooltip'
 import { PARTICIPANT } from '../../../constants'
 import SpeakingWhileMutedWarner from '../../../utils/webrtc/SpeakingWhileMutedWarner'
 import NetworkStrength2Alert from 'vue-material-design-icons/NetworkStrength2Alert'
-import { Actions, ActionSeparator, ActionButton } from '@nextcloud/vue'
+import Actions from '@nextcloud/vue/dist/Components/Actions'
+import ActionSeparator from '@nextcloud/vue/dist/Components/ActionSeparator'
+import ActionButton from '@nextcloud/vue/dist/Components/ActionButton'
```

Comment:
quite a chase through the code until I resolve the following error to these lines

```
  ● Test suite failed to run

    /home/nickv/Nextcloud/24/server/appsbabies/spreed/node_modules/emoji-mart-vue-fast/src/index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import store from './utils/store'
                                                                                             ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)
      at Object.call (node_modules/@nextcloud/vue/dist/ncvuecomponents.js:615:3949)
      at e (node_modules/@nextcloud/vue/dist/ncvuecomponents.js:1:413)
      at Module.<anonymous> (node_modules/@nextcloud/vue/dist/ncvuecomponents.js:615:1114)
```